main.go

```diff
@@ -91,6 +92,10 @@ func loadConfiguration() ([]byte, error) {
 		return nil, errors.Wrap(err, "Unable to load configuration file")
 	}
 
+	if err = envconfig.Process("", &mainCfg); err != nil {
+		return nil, errors.Wrap(err, "Unable to load environment variables")
+	}
```

Comment:
Please don't involve another configuration tool.

This project already uses `rconfig` which can be used for this. Please add the corresponding parameter into the `cfg` struct and then overwrite the mainCfg value if the attribute in `cfg` is non-empty.

Reply:
ok